api/api.py

```diff
@@ -0,0 +1,81 @@
+# Importing packages
+
+import requests
+from bs4 import BeautifulSoup
+import re
+import pandas as pd
+
+# GenscanOutput class
+
+class GenscanOutput:
+    def __init__(self, status, cds_list, intron_list, exon_list):
+        self.status = status
+        self.cds_list = cds_list
+        self.intron_list = intron_list
+        self.exon_list = exon_list
+
+# run_genscan function
+
+def run_genscan(sequence=None, sequence_file=None, organism="Vertebrate",exon_cutoff=1.00, sequence_name="", print_options='Predicted peptides only'):
+    
+    form_url = 'http://hollywood.mit.edu/cgi-bin/genscanw_py.cgi'
+
+    if sequence_file is not None:
+        sequence_file = open(sequence_file, "rb")
+        files = {'-u': sequence_file}
+
+        payload = {
+            '-o': organism,
+            '-e': exon_cutoff,
+            '-n': sequence_name,
+            '-p': print_options,
+        }
+
+        resp = requests.post(form_url, data=payload, files=files)
+        sequence_file.close()
```

Comment:
Здесь можно было открыть файл и через `with`, на лекции был не самый удачный пример
```suggestion
        with open(sequence_file, "rb") as sequence_file:
            files = {'-u': sequence_file}

            payload = {
                '-o': organism,
                '-e': exon_cutoff,
                '-n': sequence_name,
                '-p': print_options,
            }

            resp = requests.post(form_url, data=payload, files=files)
```